src/TConsole.cpp

```diff
@@ -768,7 +816,7 @@ void TConsole::closeEvent(QCloseEvent* event)
             std::tuple<bool, QString, QString> result = mpHost->saveProfile();
 
             if (std::get<0>(result) == false) {
-                QMessageBox::critical(this, tr("Couldn't save profile"), tr("Sorry, couldn't save your profile - got the following error: %1").arg(std::get<2>(result)));
+                QMessageBox::critical(this, tr("Could not save profile"), tr("Sorry, could not save your profile - the following error was found:\n%1.").arg(std::get<2>(result)));
```

Comment:
Why the change?

Reply:
Ease of comprehension by (machine) translators IMVHO - and as it is an undesired occurrence "Sorry mate I've destroyed your game data 'cos: xxxxx" doesn't carry quite the same seriousness of "Sorry, it has not been possible to save that because: xxxx" - I cannot recall where but I was always taught to avoid trivial contractions in program output (using `'` only saves one space but it does degrade readability scores as far as I understand it) and "got the following error:" is, I suspect, harder to translate than the better formed "the following error was found:" IMHO .